ivy.xml

```diff
@@ -35,7 +35,7 @@
 		<dependency org="commons-lang" name="commons-lang" rev="2.6" conf="default,standalone,maven->default" />
 		<dependency org="org.json" name="json" rev="20140107" conf="default,standalone,maven->default" />
 		<dependency org="org.apache.ant" name="ant" rev="1.9.4" conf="default" transitive="false" />
-		<dependency org="com.googlecode.java-diff-utils" name="diffutils" rev="1.3.0" conf="default,standalone,maven->default"/>
+		<dependency org="com.googlecode.java-diff-utils" name="diffutils" rev="1.3.0" conf="default,slim,standalone,maven->default"/>
```

Comment:
The interface `fitnesse.testsystems.TestResult` is independent from the subsystems either `fit` or `slim`, but the the visual presentation is in `fitnesse.testsystems.slim.HtmlTable.Cell.formatTestResult`. My patch concerns the visual representation in Slim only.